### DISCUSSION TITLE
Version 0.103 causing mypy errors

### DISCUSSION THREAD

**Discussion by deepika094** [fastapi/fastapi#10184] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Mypy source
```

### Description

Getting below error in mypy-
attribute error- attribute ‘mro’ of ‘typeinfo’ undefined

### Operating System

Windows

### FastAPI Version

0.103

### Pydantic Version

1.10.12

### Python Version

3.7

**Comment by steven-herchak** [comment]
I've encountered this as well.

It looks like it has to do with the `class Config` getting added to Example typeddict subclass when not using pydantic V2, here:
https://github.com/tiangolo/fastapi/blob/040ad986d48bb9a400de804f7f25abb856d85e1a/fastapi/openapi/models.py#L281

**Comment by Kludex** [comment, marked as answer]
FastAPI uses mypy on its code, and the pipeline is passing.
